lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -199,6 +213,136 @@ static void printNoIOOp(Operation *op, OpAsmPrinter *printer) {
   printer->printOptionalAttrDict(op->getAttrs());
 }
 
+//===----------------------------------------------------------------------===//
+// spv.AccessChainOp
+//===----------------------------------------------------------------------===//
+
+static Type extractPtrTypeFromPtrToCompositeType(ArrayRef<Value *> indices,
```

Comment:
What about naming it as `getElementPtrType`?

Reply:
Fixed.